ova/build.sh

```diff
@@ -23,6 +23,9 @@ ROOT_WORK_DIR="/go/src/github.com/vmware/dispatch/"
 ROOT_INSTALLER_DIR="${ROOT_DIR}/ova"
 ROOT_INSTALLER_WORK_DIR="${ROOT_WORK_DIR}/ova"
 
+ROOT_UI_DIR="$GOPATH/src/github.com/vmware/dispatch-ui-binaries"
```

Comment:
Does this require the user to have checked out the UI explicitly or is it done automatically earlier.

Reply:
in pipeline it automatically download the repo, and compile. But if build local, the use must have the ui repo in the right place.
Or I can add some script, to automatically clone the ui repo and build.

Reply:
Ask @kars7e's opinion also, but I would expect it to pull the repo and/or checkout origin/master when building.  Maybe allow an override with an ENV var, but otherwise the UI will always be stale when building locally unless you remember to update the UI repo before building.

Reply:
I'm not sure I like depending on the code being checked out.

Reply:
I agree with @berndtj, This actually does not fix anything, instead, it breaks stuff. what if `$GOPATH/src/github.com/vmware/dispatch-ui-binaries` does not exist? you should not depend on it.

Eventually, we want to have a pipeline that builds using versioned artifacts, but before we get there we need to ensure what we have today is functioning.